examples/hosted-delivery-function-potterdb/INSTRUCTIONS.md

```diff
@@ -0,0 +1,23 @@
+## Building your Delivery Function
+
+### Creating an app
+You can create an app using CLI using `npm run create-app-definition`. This will prompt you to enter details for your new app and organization details. Make sure your app is enabled for `Entry field` option with both Short text and JSON Object selected in the supported field types.
+
+Alternatively, you can set your app and organization details by setting them as environment variables.
```

Comment:
Personally I would find it logical to mention that they could alternatively use the web-app to create a new link (with short description where they should look). The part about environment variables is listed under the next header

Reply:
> they could alternatively use the web-app to create a new link
Wdym by this?